server.js

```diff
@@ -0,0 +1,99 @@
+var mongoose = require('mongoose');
+// Require the Express Module
+var express = require('express');
+// Create an Express App
+var app = express();
+// Require body-parser (to receive post data from clients)
+var bodyParser = require('body-parser');
+var db = mongoose.connection;
```

Comment:
This is a neat little way to talk to our DB directly, however it's still recommended to use our model variables whenever possible so we can leverage all of mongoose's great features!